src/agent/util/utils.ts

```diff
@@ -24,7 +24,8 @@ export const messages = {
   CAPTURE_BREAKPOINT_DATA: 'Error trying to capture snapshot data: ',
   INVALID_LINE_NUMBER: 'Invalid snapshot position: ',
   COULD_NOT_FIND_OUTPUT_FILE:
-      'Could not determine the output file associated with the transpiled input file'
+      'Could not determine the output file associated with the transpiled input file',
+  INSPECTOR_NOT_AVAILABLE: 'Inspector protocol only works for node 8+'
```

Comment:
What do you think about using `The V8 Inspector protocol is only available in Node 8+` instead?

Reply:
SGTM